pyproject.toml

```diff
@@ -96,13 +96,11 @@ packages = [
   'shap.actions',
   'shap.models'
 ]
-zip-safe = false
 
-[tool.setuptools.dynamic]
-version = {attr = "shap.__version__"}
-
-[tool.setuptools.package-data]
-shap = ["plots/resources/*", "cext/*"]
+[tool.setuptools_scm]
+version_file = "shap/_version.py"
```

Comment:
does this file already exist? I would have thought it is added with this PR?

Reply:
Yep, exactly.